src/common/urlUtils.ts

```diff
@@ -217,10 +224,10 @@ export function isAbsolute(_path: string): boolean {
  */
 export function urlToRegex(aPath: string) {
   const absolutePath = fileUrlToAbsolutePath(aPath);
-
-  aPath = escapeRegexSpecialChars(aPath);
+  const eatQueryString = '(\\?.*)?';
+  aPath = escapeRegexSpecialChars(aPath) + eatQueryString;
   if (absolutePath) {
-    aPath += `|${escapeRegexSpecialChars(absolutePath)}`;
+    aPath += `|${escapeRegexSpecialChars(absolutePath)}${eatQueryString}`;
```

Comment:
This seems unnecessary because the regex will still match up to the `?`

Reply:
It seems that Chrome implicitly wraps the provided regex in `^$` when I tested it earlier

Reply:
actually, nevermind! I was probably running into something else at the time